suite-native/graph/src/components/Graph.tsx

```diff
@@ -76,6 +78,16 @@ export const Graph = <TGraphPoint extends EnhancedGraphPoint>({
     onTryAgain,
     error,
 }: GraphProps<TGraphPoint>) => {
+    const addToastNotification = useSetAtom(addToastNotificationAtom);
```

Comment:
Please create hook that will wrap this so you don't need to import both `jotai` and `addToastNotificationAtom`. Something like `const { showToast } = useToast();` Also please used that named export through object for `showToast` so it must be named everywhere same.

Reply:
9c3195cce